test/e2e/csv_e2e_test.go

```diff
@@ -130,7 +131,43 @@ var _ = Describe("ClusterServiceVersion", func() {
 				}
 
 				return original.Status.LastUpdateTime.Equal(copy.Status.LastUpdateTime), nil
-			}).Should(BeTrue(), "Change to status of copy should have been reverted")
+			}).Should(BeFalse(), "Change to status of copy is not reverted")
+		})
+		It("is synchronized with the original CSV based on spec changes to the original", func() {
+			Eventually(func() error {
+				key, err := client.ObjectKeyFromObject(&original)
+				if err != nil {
+					return err
+				}
+
+				key.Namespace = target.Name
+				if err := ctx.Ctx().Client().Get(context.Background(), key, &original); err != nil {
+					return err
+				}
+
+				// add extra port (443) to the deployment spec on the CSV
+				original.Spec.InstallStrategy.StrategySpec.DeploymentSpecs[0].Spec.Template.Spec.Containers[0].Ports = []corev1.ContainerPort{{ContainerPort: 80}, {ContainerPort: 443}}
```

Comment:
It almost feels like we should be eliding everything but the metadata useful to operator consumers. For example, the `InstallStrategy` is irrelevant to operator consumers. In general, I think we need to keep everything used directly by Console.

Reply:
This is a good point but beyond the scope of what we originally envisioned when we came up with the bugfix, which was just the minimal "this CSV is copied" properties. Is there an easy way to determine all the metadata the console looks at for a CSV?